src/ray/gcs/gcs_server/gcs_placement_group_scheduler.h

```diff
@@ -158,6 +175,10 @@ class GcsPlacementGroupScheduler : public GcsPlacementGroupSchedulerInterface {
   /// bundles when a node is dead.
   absl::flat_hash_map<ClientID, std::vector<std::shared_ptr<BundleSpecification>>>
       node_to_leased_bundles_;
+  /// Map from placement group ID to the map of bundles. When we reschedule bundles, we
+  /// can get the location of other bundles from here.
+  absl::flat_hash_map<PlacementGroupID, absl::flat_hash_map<int64_t, ClientID>>
```

Comment:
If I understand correctly, this is just the reverse index of node_to_leased_bundles_, is that right? Maybe this can be added as a comment.

Reply:
Btw, I have almost the same map with a little different types. https://github.com/ray-project/ray/pull/10063 We should probably use the same type index for this.

Reply:
Maybe we can merge this PR first and I can handle differences in my PR next week?

Reply:
We could also pull this out into a separate data structure to avoid errors dealing with the multiple maps, but maybe that is better done in a separate PR.

Reply:
@ericl Definitely +1. I was actually trying to do that in my PR, but the class was too shallow, so I reverted. Maybe after we merge those 2 PRs, it is a good time to do that. (We can make a class like `leasingContext`).